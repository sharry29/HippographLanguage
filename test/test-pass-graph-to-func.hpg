int main() {
	graph<int, string> g = [1 -("a")- 2];
	add_node_remotely(g; 3);
	add_node_remotely(g; 4);
	add_node_remotely(g; 5);
	g.print();
	return 0;
}

void add_node_remotely(graph<int, string> g; int label) {
	node<int> n = label;
	g.set_node(n);
}