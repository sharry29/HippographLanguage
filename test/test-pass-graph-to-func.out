1:null -> [2:null (a)]
2:null -> [1:null (a)]
3:null -> []
4:null -> []
5:null -> []